appeals/web/src/server/appeals/appeal-details/interested-party-comments/view-and-review/page-components/reject.mapper.js

```diff
@@ -29,21 +28,23 @@ export function rejectInterestedPartyCommentPage(appealDetails, comment) {
 }
 
 /**
+ * @param {import('got').Got} apiClient
  * @param {Appeal} appealDetails
  * @param {Representation} comment
- * @returns {PageContent}
+ * @returns {Promise<PageContent>}
  * */
-export function rejectAllowResubmitPage(appealDetails, comment) {
+export async function rejectAllowResubmitPage(apiClient, appealDetails, comment) {
 	const shortReference = appealShortReference(appealDetails.appealReference);
-	const deadline = dateISOStringToDisplayDate(addBusinessDays(new Date(), 7).toISOString());
+	const deadline = await addBusinessDays(apiClient, new Date(), 7);
+	const deadlineString = dateISOStringToDisplayDate(deadline.toISOString());
```

Comment:
I mentioned in a chat this could probably fail tomorrow. The deadline in the snapshot will never change...